lib/MusicBrainz/Server/Data/URL.pm

```diff
@@ -50,8 +50,6 @@ my %URL_SPECIALIZATIONS = (
     'DAHR'                => qr{^https?://adp\.library\.ucsb\.edu/}i,
     'Dailymotion'         => qr{^https?://(?:www\.)?dailymotion\.com/}i,
     'DanceDB'             => qr{^https?://(?:www\.)?tedcrane\.com/DanceDB/}i,
-    # Temporarily grey Decoda out as it has been reported to be compromised
-    # 'Decoda'              => qr{^https?://(?:www\.)?decoda\.com/}i,
```

Comment:
Dropping this makes Decoda URLs no longer greyed out under entity’s “Relationships” tab.

Reply:
I can drop all existing decoda rels before merging this (I just finished the previous MBBE bot run anyway), so if we block stuff as suggested below this should be fine.

Reply:
In any case, there will be a period of time during which Decoda URLs can still be entered in production and seen in beta. Let’s be safe and keep these greyed out in next server release, and then drop this code as well as the file `lib/MusicBrainz/Server/Entity/URL/Decoda.pm` (missing from this PR) only after this patch has been deployed in production.

Reply:
Thanks for having dropped `lib/MusicBrainz/Server/Entity/URL/Decoda.pm` (which was the real issue here), and still greying out  the URL in JavaScript code.